modules/module-extras.php

```diff
@@ -22,6 +22,7 @@
 	'seo-tools/jetpack-seo-utils.php',
 	'seo-tools/jetpack-seo-titles.php',
 	'seo-tools/jetpack-seo-posts.php',
+	'unified-importer/unified-importer.php',
```

Comment:
Do you mean for this new module to be available to everyone using the Jetpack plugin, whether they've connected the plugin to WordPress.com or not? If the module relies on any WordPress.com components (like info about the shadow site), you may have to consider that before loading the file (see how the SEO tools do it below).

Reply:
@jeherve Thanks for jumping in!

Only some features in this module will need a connection to WordPress.com, mainly the url importer... so my current thinking is that we'll always show the UI and "gracefully degrade" if not connected, showing only file importer (and maybe a nudge to connect to enable url imports)